README.md

```diff
@@ -45,7 +45,6 @@ EmojiPicker(
         indicatorColor: Colors.blue,
         iconColor: Colors.grey,
         iconColorSelected: Colors.blue,
-        progressIndicatorColor: Colors.blue,
```

Comment:
I did remove it from config. 
When I feel particularly fancy, I try to make loading indicators appear with 0.5 seconds delay, because blinking indicator doesn't make it look prettier. Also, if you aim for best platform experience, you would have to switch between `CircularProgressIndicator` and `CupertinoActivityIndicator` depending on the platform. I would suggest to leave it `SizedBox` by default and add a widget host can provide (I suggest doing empty by default, because with `CircularProgressIndicator` you would have to bring color option back which bloats config even more).
Let me know if you agree.

Reply:
Good point about different platform experience ! 
Optional custom Widget which is empty by default would be nice, but I'm fine with addressing that later unless you want to add it right away 👍

Reply:
Just noticed, we can also add `Custom-Emoji-Font support` to `## Key features`

Reply:
No problem, I've added progress indicator widget in config. Do you mind updating README with key features, new example, etc. yourself?

Reply:
Awesome, thank you ! Yeah that's alright 👍 I will add it myself